chart/monitoring/ClusterSelector.vue

```diff
@@ -153,6 +155,8 @@ export default {
         }
       } else {
         this.setClusterTypeEnabledValues([configKeys, true]);
+        this.$set(this.value.prometheusOperator.kubeletService, 'enabled', false);
```

Comment:
Why does `kubeletService` need to be disabled? Does RKE already create a kubelet service?

Reply:
Technically we could leave it enabled, but as we're not using a ServiceMonitor with the pushprox we don't need the service to manage the endpoints.  I don't believe RKE creates one, as it's used purely for the ServiceMonitor.

Reply:
I see your point. Resolving this comment.

Also, to clarify, we do use a [ServiceMonitor](https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-pushprox/charts/templates/pushprox-servicemonitor.yaml#L3) for pushprox, but you are right that this ServiceMonitor already comes with a [Service](https://github.com/rancher/charts/blob/eec20be500c294a87b7ce3e2a1bdbd1624476fa5/packages/rancher-pushprox/charts/templates/pushprox-servicemonitor.yaml#L27) pointing to the pushprox-client DaemonSet. So we don't need the additional service pointing to Kubelet.